west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.9.0-rc1
+      revision: pull/650/head
```

Comment:
I've merged the nrfxlib PR, but the manifest will be updated in a different PR to avoid multiple updates. Please drop the `west.yml` change from the PR so we can merge the doc changes only.

Reply:
Hi @rlubos : I have fixed it.